src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -116,3 +118,25 @@ export const selectUpdatePlantingCompleted = (state: RootState, requestId: strin
 
 export const selectUpdatePlantingsCompleted = (state: RootState, requestId: string) =>
   (state.updatePlantingsCompleted as any)[requestId];
+
+export const selectZonesHaveStatistics = createSelector(
+  [
+    (state: RootState, zoneIds?: number[]) => selectLatestObservation(state, -1),
```

Comment:
This will return the latest observation across all planting sites.

You may need to pass in a record of  Record<siteId, list of zone ids>
and then maybe do a select for each of those site Ids.